Cargo.toml

```diff
@@ -4,46 +4,47 @@ members = ["crates/*"]
 
 [workspace.dependencies]
 
-Inflector             = "0.11.4"
-anyhow                = "1.0.68"
-assert_fs             = "1.0"
-async-graphql         = "4.0.9"
-async-graphql-poem    = "4.0.9"
-async-stream          = "0.3.3"
-async-trait           = "0.1.61"
-backoff               = { version = "0.4.0", features = ["futures", "tokio"] }
-base64                = "0.21"
-bytes                 = "1.3.0"
-cached                = "0.42"
-chrono                = "0.4.19"
-clap                  = "3.2.1"  # opactl: version = "4.1.1"
-clap_complete         = "3.1.1"
-clap_generate         = "3.0.3"
-colored_json          = "3.0.1"
-criterion             = { version = "0.4.0", features = ["async_futures", "async_tokio"] }
-crossbeam             = "0.8.1"
-custom_error          = "1.9.2"
-derivative            = "2.2.0"
-diesel                = { version = "2.0.0-rc.0", features = [
+Inflector = "0.11.4"
+anyhow = "1.0.68"
+assert_fs = "1.0"
+async-graphql = "5.0.6"
+async-graphql-poem = "5.0.6"
+async-stream = "0.3.3"
+async-trait = "0.1.61"
+backoff = { version = "0.4.0", features = ["futures", "tokio"] }
+base64 = "0.21"
+bytes = "1.3.0"
+cached = "0.42"
+chrono = "0.4.19"
+clap = "3.2.1" # opactl: version = "4.1.1"
+clap_complete = "3.1.1"
+clap_generate = "3.0.3"
+colored_json = "3.0.1"
+criterion = { version = "0.4.0", features = ["async_futures", "async_tokio"] }
+crossbeam = "0.8.1"
+custom_error = "1.9.2"
+derivative = "2.2.0"
+diesel = { version = "2.0.0-rc.0", features = [
   "postgres",
   "uuid",
   "chrono",
   "r2d2",
 ] }
-diesel_migrations     = { version = "2.0.0-rc.0", features = ["postgres"] }
-dotenvy               = "0.15"
-futures               = "0.3.21"
-genco                 = "0.16.1"
-glob                  = "0.3.0"
-hashbrown             = "0.13"
-hex                   = "0.4.3"
-insta                 = { version = "1.26.0", features = ["redactions", "toml"] }
-iref                  = "2.1.2"
-iref-enum             = "2.0.0"
-json-ld               = "0.14"
-jsonschema            = "0.17.0"
-jwtk                  = { version = "0.2.4", features = ["remote-jwks"] }
-k256                  = { version = "0.11.3", features = [
+diesel_migrations = { version = "2.0.0-rc.0", features = ["postgres"] }
+dotenvy = "0.15"
+futures = "0.3.21"
+genco = "0.16.1"
+glob = "0.3.0"
+hashbrown = "0.13"
+hex = "0.4.3"
+insta = { version = "1.26.0", features = ["redactions", "toml"] }
+iref = "2.2"
+iref-enum = "2.1"
+json-ld = { version = "0.14" }
+json-syntax = { version = "*", features = ["serde", "serde_json"] }
```

Comment:
Can this now be removed?

Reply:
No, this is how the feature is enabled in a sub dependency

Reply:
How come we need it now but didn't before?

Reply:
Unknown, but this is the only way to activate features in transitive dependencies I know of

Reply:
Thanks. Most curious that we didn't need this before this PR, do let me know if you ever figure out what the relevant trigger was!